src/main/java/org/example/model/House.java

```diff
@@ -0,0 +1,53 @@
+package org.example.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class House {
+
+    private List<Card> hand;
+    private int score;
+
+    public House() {
+        hand = new ArrayList<>();
+    }
+
+    public void addCardToHand(Card card) {
+        hand.add(card);
+    }
+
+    public int countScore(List<Card> hand) {
+        return (int) sumValuesOfCards(hand);
+    }
+
+    private long sumValuesOfCards(List<Card> cards) {
+        return cards.stream().mapToInt(this::numberValueFromCard).sum();
```

Comment:
Unnecessary mapping to int in countScore method can be fixed here by using collect method

Reply:
I corrected it, but I don't understand why this solution is correct.